client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -89,12 +91,14 @@ class BusinessDetails extends Component {
 			currency: currency.code,
 			revenue,
 			used_platform: otherPlatform,
+			used_platform_name: otherPlatformName,
 			install_facebook: values[ 'facebook-for-woocommerce' ],
 			install_mailchimp: values[ 'mailchimp-for-woocommerce' ],
 		} );
 
 		const _updates = {
 			other_platform: otherPlatform,
+			other_platform_name: otherPlatformName,
```

Comment:
Maybe this should set the other platform name to an empty string here instead of the added logic below.

```suggestion
			other_platform_name: otherPlatform === 'other' ? otherPlatformName : '',
```

Reply:
You are totally right! I will change it.
Thank you :slightly_smiling_face: